tests/post/steps/test_salt_ssh.py

```diff
@@ -0,0 +1,43 @@
+import json
+
+import pytest
+from pytest_bdd import scenario, then, parsers
+
+from tests import utils
+
+
+@scenario(
+    "../features/salt_ssh.feature", "salt-ssh state work on every non-bootstrap nodes"
+)
+def test_salt_ssh_state(host):
+    pass
+
+
+@then(
+    parsers.parse("we are able to run '{state}' using salt-ssh on non-bootstrap nodes")
```

Comment:
Could be a variable, but... hey, for now I think it will be enough. So, feel free to ignore.
```suggestion
    parsers.parse("we are able to run '{state}' using salt-ssh on non-{role} nodes")
```

Reply:
I will do it since build just started